webpack/components/AnsibleHostDetail/AnsibleHostDetail.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import AnsibleHostDetail from './index';
```

Comment:
no need to add index name to import
```suggestion
import AnsibleHostDetail from './';
```